backend/core/tests/test_detection.py

```diff
@@ -71,6 +71,14 @@ def setUp(self) -> None:
                 "mitigation": ["manual"],
                 "community_annotations": [],
             },
+            {
```

Comment:
Do you think it would make more sense to have the rules defined in each test instead of setup?

Else we might loose the "ball" 🙄

Reply:
Will be addressed in separate issue #441